### DISCUSSION TITLE
How to make multipage editor?

### DISCUSSION THREAD

**Discussion by Bhaliya-jigar** [prevwong/craft.js#192] [Q&A]
How can i make multipage editor with Craft JS? Should i have to create separate `<Editor>` for each page ? i have started with [this ](https://github.com/prevwong/craft.js/tree/master/packages/examples) example.  i have tried to add multiple `<Frame>` within one  `<Editor>` without any luck.

**Comment by magnus-trent** [comment]
I'm working on something similar. Basically I believe Craft is a page builder alone. So making it a multi page/ website builder you'd have to do that on your own implementation with Craft as the interactive tool for editing each page. If that makes sense.

**Reply by ankri** [reply]
Yes. In my project I am using one `<Editor>` and `<Frame>` per page.

**Comment by yusuf-mutlu** [comment]
Does anyone have any example code about this?
